openapi/templates/index.js

```diff
@@ -494,3 +495,8 @@ function isUrnGrantType(grantType) {
 function prefixUrnGrantType(grantType) {
   return URN_PREFIX + URN_SUFFIXES[grantType];
 }
+
+const RESPONSE_MODELS_WITH_CUSTOM_ATTRIBUTES = ['User', 'Group', 'UserSchema', 'GroupSchema'];
```

Comment:
Applying logic to these models which have been reported to have issues with custom attributes